library/cohesity_job.py

```diff
@@ -582,7 +582,7 @@ def update_job(module, job_details, update_source_ids):
                    "user-agent": "Ansible-v2.2.0"}
         payload = job_details.copy()
         del payload['token']
-        if module.params.get('environment') == 'PhysicalFiles':
+        if module.params.get('environment') == 'PhysicalFiles' and module.params.get('delete_sources') == False:
```

Comment:
Update the documentation for this. And if key doesn't exists default it to false ?

Reply:
yeah